src/flick/show/serializers.py

```diff
@@ -15,16 +16,19 @@ class ShowSerializer(serializers.ModelSerializer):
     comments = serializers.SerializerMethodField(method_name="get_friends_and_user_comments")
     friends_rating = serializers.SerializerMethodField(method_name="calculate_friends_rating")
     user_rating = serializers.SerializerMethodField(method_name="get_user_rating")
+    providers = ProviderSerializer(read_only=True, many=True)
 
     class Meta:
         model = Show
         fields = (
             "id",
+            "ext_api_id",
```

Comment:
what's that

Reply:
nvm you mentioned it in the PR

Reply:
u can ignore it! it's for backend to debug hehe (it's our api provider's id assignment to the show)